Form Deletion

supriyabhargava opened the pull request:
Requirements and mockups for Form deletion

kamilmusial commented:
@supriyabhargava What would happen if user decide to use one of the previous versions of landing page, where form is still present?

dew326 commented on eZ Platform/Studio/Form Builder/Deleting Form/deleting_a_form.md:
```diff
@@ -0,0 +1,92 @@
+### User story 1 - As a User, I should have the ability to select a form in the Form Manager for deletion
+*Changes to the current implementation include: <br /> 1. adding a radio button for each form in the table and <br /> 2. adding a 'Delete' button at the top right corner of the table. <br /><br /> By default (if forms exists), then no form will be selected and the 'Delete' button will be disabled.*
+
+***Please refer to the attached image 'FormManagerwithDeleteOption.png'***
```
You can embed the image into the markdown, it improves the readability.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images

So for this image, it can look like: `![](FormManagerwithDeleteOption.png)`

supriyabhargava replied:
thanks for the tip

DavidLiedle commented:
@kamilmusial @supriyabhargava - that raises an excellent question. Perhaps in that case, when a previous version which had included the form is resurrected, in the editor's view it may display a large X'd-out region with a notification/warning, and prevent publishing until edited to remove?

supriyabhargava commented:
@kamilmusial - We should keep the behavior consistent with the other blocks. For e.g., On the landing page (Version 1), we add Brooklyn, New York to the Main Story block. A few versions later (landing page), I changed the story to Amsterdam and deleted the content (Brooklyn, New York). When I access the landing page V1, then in the view mode I will not see the Main story block while in the edit mode the block will appear empty. Similarly, for Form Builder block, in the view mode, the block will not appear on the landing page while in the edit mode the user will see an empty block.
